test/runtime/privileged_tests.go

```diff
@@ -29,7 +29,7 @@ import (
 const (
 	// The privileged unit tests can take more than 4 minutes, the default
 	// timeout for helper commands.
-	privilegedUnitTestTimeout = 8 * time.Minute
+	privilegedUnitTestTimeout = 16 * time.Minute
```

Comment:
:scream:

Reply:
:shrug: It unlikely to increase the test duration on average since we never get "stuck" in unit tests and outliers with high duration are rare (although annoying).

> I wonder what would be the best way to track these spikes in execution, so we don't just increase timeouts ad infinitum.

@nebril I agree, but I'm unsure what to track next (it's not due to CPU frequency issues). Definitely not planning to increase the timeout beyond 16min :laughing: